### PR TITLE
0.2.3 Release Proposal

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Cloud Diagnostics Common ChangeLog
+
+## 2016-04-18, Version 0.2.3 (Experimental), @matthewloring
+
+### Commits
+
+* [[`1c4a9ff72f`](https://github.com/GoogleCloudPlatform/cloud-diagnostics-common-nodejs/commit/1c4a9ff72f)] - Get travis running (Matt Loring) 
+* [[`93811b80aa`](https://github.com/GoogleCloudPlatform/cloud-diagnostics-common-nodejs/commit/93811b80aa)] - GCLOUD_PROJECT instead of GCLOUD_PROJECT_NUM (Matt Loring)
```
Comment (ofrobots) [inline CHANGELOG.md]: This change merits more than a patch-level bump.

**Comment by ofrobots** [comment]
LGTM w/ nit.

File: CHANGELOG.md (hunk 1 above)
Comment (ofrobots) [inline CHANGELOG.md]: Upon second look, that is a message change. Looks fine as it is.

**Comment by ofrobots** [comment]
LGTM no nits.

**Pull request merged**